scripts/zones/Aht_Urhgan_Whitegate/npcs/Ghatsad.lua

```diff
@@ -360,169 +226,169 @@ function onTrigger(player,npc)
-        elseif (attachments == 2) then
-            player:startEvent(622, 0, 3, 0, 0, 0, 2187, 1);
+    elseif NoStringsAttached == QUEST_COMPLETED and attachmentStatus == 4 then
+        if attachments == 0 then
+            player:startEvent(622, 0, 3, 0, 0, 0, 2185, 3)
+        elseif attachments == 1 then
+            player:startEvent(622, 0, 3, 0, 0, 0, 2186, 3)
+        elseif attachments == 2 then
+            player:startEvent(622, 0, 3, 0, 0, 0, 2187, 1)
         end
-    elseif (NoStringsAttached == QUEST_COMPLETED and attachmentStatus >= 5 and attachmentStatus <= 7) then
-        if (attachments == 0) then
-            player:startEvent(624, 0, 0, 0, 0, 0, 2185, 3);
-        elseif (attachments == 1) then
-            player:startEvent(624, 0, 0, 0, 0, 0, 2186, 3);
-        elseif (attachments == 2) then
-            player:startEvent(624, 0, 0, 0, 0, 0, 2187, 1);
+    elseif NoStringsAttached == QUEST_COMPLETED and attachmentStatus >= 5 and attachmentStatus <= 7 then
+        if attachments == 0 then
+            player:startEvent(624, 0, 0, 0, 0, 0, 2185, 3)
+        elseif attachments == 1 then
+            player:startEvent(624, 0, 0, 0, 0, 0, 2186, 3)
+        elseif attachments == 2 then
+            player:startEvent(624, 0, 0, 0, 0, 0, 2187, 1)
         end
-    elseif (NoStringsAttached == QUEST_COMPLETED and attachmentStatus == 11 and attachments == 3) then
-        player:startEventString(900, automatonName, automatonName, automatonName, automatonName, 0, 0, 1, 0, 0, 2186, 7);
-    elseif (NoStringsAttached == QUEST_COMPLETED and (attachmentStatus == 12 or attachmentStatus == 13) and attachmentReady == false) then
-        player:startEvent(903, attachmentWait, 1);
-    elseif (NoStringsAttached == QUEST_COMPLETED and (attachmentStatus == 12 or attachmentStatus == 13) and attachmentReady == true and attachmentWait > 0) then
-        player:startEvent(903, attachmentWait, 0);
-    elseif (NoStringsAttached == QUEST_COMPLETED and (attachmentStatus == 12 or attachmentStatus == 13) and attachmentReady == true and attachmentWait == 0) then
-        player:startEvent(905, attachmentStatus-12);
-    elseif (NoStringsAttached == QUEST_COMPLETED and attachmentStatus == 14) then
-        if (unlockedAttachments == 30) then
-            player:startEventString(901, automatonName, automatonName, automatonName, automatonName, 1, 0, 1, 0, 0, 2186, 4);
-        elseif (unlockedAttachments == 46) then
-            player:startEventString(901, automatonName, automatonName, automatonName, automatonName, 0, 0, 1, 0, 0, 2186, 4);
+    elseif NoStringsAttached == QUEST_COMPLETED and attachmentStatus == 11 and attachments == 3 then
+        player:startEventString(900, automatonName, automatonName, automatonName, automatonName, 0, 0, 1, 0, 0, 2186, 2)
+    elseif NoStringsAttached == QUEST_COMPLETED and (attachmentStatus == 12 or attachmentStatus == 13) and attachmentReady == false then
```

Comment:
can group stuff below this under a single check for quest completed and attachment status, then the rest can be checked